src/lib/operators/table_scan/column_is_null_table_scan_impl.cpp

```diff
@@ -151,6 +255,32 @@ bool ColumnIsNullTableScanImpl::_matches_none(const BaseValueSegment& segment) c
   }
 }
 
+bool ColumnIsNullTableScanImpl::_matches_all(const BaseDictionarySegment& segment) const {
```

Comment:
note here: for such shortcuts, do not forget to increase their respective counters
https://github.com/hyrise/hyrise/blob/5306d23c855bd24bc112295f582064f16a7a5a8c/src/lib/operators/table_scan/abstract_table_scan_impl.hpp#L33-L35

Reply:
For other scan implementations, this was done in #2283